main/webapp/modules/core/externals/suggest/suggest-4_3a.js

```diff
@@ -1089,71 +1064,6 @@
     _init: function() {
       var self = this,
           o = this.options;
-      if (o.flyout_service_url == null) {
-        o.flyout_service_url = o.service_url;
-      }
-      this.flyout_url = o.flyout_service_url;
-      if (o.flyout_service_path) {
-          this.flyout_url += o.flyout_service_path;
-      }
-      // set api key for flyout service (search)
-      this.flyout_url = this.flyout_url.replace(/\$\{key\}/g, o.key);
-      if (o.flyout_image_service_url == null) {
-        o.flyout_image_service_url = o.service_url;
-      }
-      this.flyout_image_url = o.flyout_image_service_url;
-      if (o.flyout_image_service_path) {
-          this.flyout_image_url += o.flyout_image_service_path;
-      }
-      // set api key for image api
-      this.flyout_image_url = this.flyout_image_url.replace(/\$\{key\}/g, o.key);
-
-      if (!$.suggest.cache) {
-        $.suggest.cache = {};
-      }
```

Comment:
Those three lines above likely need to be kept, I think the cache is still used by the suggest service independently of the flyout.